### PR TITLE
Fix the coding style of Ballerina implementation and tests

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2255

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by niveathika — APPROVED** [review]

File: ballerina/tests/bytes_io.bal [hunk 1]
```diff
@@ -142,38 +96,31 @@ isolated function testFileWriteBytesFromStreamUsingIntermediateFile() returns Er
...
-        if (returnedString is string) {
-            test:assertEquals(returnedString, expectedString);
-        } else {
-            test:assertFail(msg = returnedString.message());
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+    string|error returnedString = langstring:fromBytes(byteArr);
```
Comment (daneshk) [inline ballerina/tests/bytes_io.bal:113]:
```suggestion
    string returnedString = check langstring:fromBytes(byteArr);
    test:assertEquals(returnedString, expectedString);
```

File: ballerina/tests/bytes_io.bal [hunk 2]
```diff
@@ -182,67 +129,47 @@ function testFileWriteBytesFromStream() {
...
-            test:assertEquals(returnedString, expectedString);
-        } else {
-            test:assertFail(msg = returnedString.message());
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+
+    string|error returnedString = langstring:fromBytes(byteArr);
```
Comment (daneshk) [inline ballerina/tests/bytes_io.bal:147]: Same as above

File: ballerina/tests/bytes_io.bal [hunk 3]
```diff
@@ -271,107 +190,63 @@ function testFileChannelReadBytesAsStream() returns Error? {
...
-        } else {
-            test:assertFail(msg = result.message());
+    ReadableByteChannel fileOpenResult = check openReadableFile(filePath);
+    stream<Block, Error?> result = check channelReadBlocksAsStream(fileOpenResult, 2);
+    check result.forEach(function(Block val) {
+        foreach byte b in val {
+            byteArr.push(b);
         }
+    });
+    string|error returnedString = langstring:fromBytes(byteArr);
```
Comment (daneshk) [inline ballerina/tests/bytes_io.bal:200]:
```suggestion
    string returnedString = check langstring:fromBytes(byteArr);
```

Comment (BuddhiWathsala) [reply ballerina/tests/bytes_io.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4

File: ballerina/tests/bytes_io.bal (hunk 2 above)
Comment (BuddhiWathsala) [reply ballerina/tests/bytes_io.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4

File: ballerina/tests/bytes_io.bal (hunk 1 above)
Comment (BuddhiWathsala) [reply ballerina/tests/bytes_io.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/420/commits/917a8e482c0c914bc2ca9970fe31f935023fbdf4

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**